addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -373,7 +373,7 @@ const threadPatch = {
      * @returns {import("models").ChannelMember[]}
      */
     get membersThatCanSeen() {
-        return this.channel.channel_member_ids;
+        return this.channel?.channel_member_ids ?? [];
```

Comment:
This is a classical flow: agent testing their own live chat config. We should ensure such common flows are properly tested, especially since a lot of working is ongoing with the channel model.